compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -600,22 +600,24 @@ class ComposeScene internal constructor(
     }
 
     private fun processPress(event: PointerInputEvent) {
-        val owner = reversedOwners {
-            for (owner in it) {
-                if (owner.isHovered(event)) {
-                    // Stop once the position of in bounds of the owner
-                    return@reversedOwners owner
-                }
-                owner.onClickOutside?.invoke()
-                if (owner == focusedOwner) {
-                    // Stop if it's in focus, do not pass the event to hovered owner
-                    return@processPress
-                }
+        val previousPressOwner = pressOwner
+        if (previousPressOwner != null) {
+            previousPressOwner.processPointerInput(event)
+            return
+        }
```

Comment:
Are you sure that's what we want to do? It means events will continue to be sent to the very first pressed owner, even if none current touches started within it:
1. Touch[id=1] on owner1 -> sent to owner1
2. Touch[id=2] on owner2 -> sent to owner1
3. Release[id=1] -> sent to owner1
4. Touch[id=3] -> sent to owner1

Reply:
Yes. The logic here is (and was, but with implementation issues) that _the gesture_ starts with first touch and ends until last one. If we start to interact with owner1, we should sent all events to it for correctly handling all kinds of gestures

Reply:
That logic makes sense to me on the face of it, but without a real investigation, I don't have certainty. Also, if we decide that that's a principle for us, it's worth documenting it.

Reply:
Added a video from Android with expected behavior to PR description